source/trackers.json

```diff
@@ -1072,6 +1072,7 @@
         "adjust.net.in": "adjust",
         "adjust.world": "adjust",
         "apptrace.com": "adjust",
+        "adobe.io": "adobe_experience_cloud",
```

Comment:
```
"adobe_experience_cloud": {
            "name": "Adobe Experience Cloud",
            "categoryId": 6,
            "url": "https://www.adobe.com/experience-cloud.html",
            "companyId": "adobe"
        },
```
Has wrong website https://business.adobe.com/

`adobe.io` - redirects to https://developer.adobe.com/ , maybe it should be in separate tracker